juicebox_cli/cli.py

```diff
@@ -67,18 +56,17 @@ def login(ctx, username, env):
 @click.option('--netrc', default=None)
 @click.option('--job')
 @click.option('--app', default=None)
-@click.option('--env', envvar='JB_ENV', default='prod')
+@click.option('--endpoint', required=True)
```

Comment:
What would the value for this environment variable be? And how would it be different for every client install?

Reply:
Sorry, I think I failed to give enough context.

To answer your question: it would be whatever you normally pass to `--endpoint`.

click has support for automatically reading environment variables to supply values to the `click.option` parameters. That's what the `envvar` parameter does. We previously had `JB_ENV` to supply the equivalent of `--env`, and it seems like it would be equally convenient to do the same for the endpoint, with a `JB_ENDPOINT` parameter. So if someone sets this `JB_ENDPOINT` environment variable, then they don't need to worry about passing `--endpoint` to `juice login` and `juice upload`.

Reply:
Aaaaah ok, I got it now 👍